nuc_morph_analysis/analyses/volume/FigS10_workflow.py

```diff
@@ -195,9 +195,9 @@
     ax.set_xlabel('Movie time (hr)')
 
 
-    savename = save_dir / f"S10_D-volume_dips_over_time_all_colonies-{ycol}-{threshold}"
```

Comment:
This was going to make a path like `save_dir / save_dir / f"S10_D-volume_dips_over_time_all_colonies-{ycol}-{threshold}"` so I removed one of the duplicate `save_dir`s.